fdbserver/ptxn/test/Driver.actor.cpp

```diff
@@ -68,6 +68,27 @@ TestDriverOptions::TestDriverOptions(const UnitTestParameters& params)
     transferModel(static_cast<MessageTransferModel>(
         params.getInt("messageTransferModel").orDefault(static_cast<int>(DEFAULT_MESSAGE_TRANSFER_MODEL)))) {}
 
+namespace {
+// Initialize the server DB info in TestDriverContext, this should be called after all other parts of TestDriverContext
+// get initialized.
+void initServerDBInfo(std::shared_ptr<TestDriverContext> pContext) {
+	auto& dbInfo = pContext->dbInfo;
+
+	dbInfo.id = randomUID();
+	// FIXME fake cluster controller?
+	// dbInfo.clusterInterface = ??
+	// FIXME ClientDBINfo?
+	// dbInfo.client = ??;
+	dbInfo.distributor.reset();
+	dbInfo.master = *pContext->sequencerInterface;
+	dbInfo.ratekeeper.reset();
+	std::transform(std::begin(pContext->resolverInterfaces),
+	               std::end(pContext->resolverInterfaces),
+	               std::back_inserter(dbInfo.resolvers),
+	               [](const auto& ptr) -> auto { return *ptr; });
+}
```

Comment:
Can you add `LogSystemConfig` here? For now, you could just add known tlog interfaces to `LogSystemConfig::tLogs[0]`, i.e., the primary DC.

Reply:
This could be hard, as 
```
LogSystemConfig::tLogs::tLogs
```
is using the old `TLogInterface`. I do not have a straight idea about solving it right now.